apps/andi/lib/andi_web/live/shared_components/data_dictionary/data_dictionary_field_editor.ex

```diff
@@ -26,6 +26,12 @@ defmodule AndiWeb.DataDictionary.FieldEditor do
         <%= hidden_input(@form, :bread_crumb) %>
         <%= hidden_input(@form, :dataset_id) %>
         <%= hidden_input(@form, :ingestion_id) %>
+        <%= hidden_input(@form, :name) %>
+        <%= hidden_input(@form, :type) %>
```

Comment:
Why these hidden inputs?

Reply:
because disabled fields don't push their values onward to the submit. Because of how the changesets going into this are set up, only really the changes get populated. This is to prevent the fields from emptying out if you try to save (they would still be there, just the UI would show empty)

Reply:
Oh, nice catch on that additional issue. Didn't catch that when I disabled the schema

Reply:
technically, readonly is the way to go through that it allows tabbing through and submission, but not editing (but I think there was a specific requirement to not let it tabbed through, so that's y I deemed to go with this instead)